dynamiq/prompts/prompts.py

```diff
@@ -56,11 +56,13 @@ class Message(BaseModel):
         content (str): The content of the message.
         role (MessageRole): The role of the message sender.
         metadata (dict | None): Additional metadata for the message, default is None.
+        prefix (bool | None): Whether the message should be prefixed, default is None.
```

Comment:
What this prefix mean, and what purpose. Please add example in docstring or comments